.github/workflows/ruby.yml

```
@@ -6,82 +6,16 @@ on:
     branches:
       - master
 
-env:
-  RAILS_ENV: test
-  DATABASE_URL: postgresql://postgres:@localhost/test
-  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
-
 jobs:
   rubocop:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - run: sudo apt-get update
-      - run: sudo apt-get install libyaml-dev
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: 2.7
-          bundler-cache: true
-      - name: Run rubocop
-        if: github.event_name != 'push'
-        run: bundle exec rubocop --format github
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+    with:
+      command: bundle exec rubocop --format github
```

While we're changing things:
```suggestion
      command: bundle exec rubocop --parallel --format github
```